source/docs/software/pathplanning/system-identification/viewing-diagnostics.rst

```diff
@@ -1,53 +1,63 @@
+The velocity time domain plots contain three sets of data: Raw Data, Filtered Data, and Simulation. The Raw Data is the recorded data from your robot, the Filtered Data is the data after a median filter has been applied to the data, and the Simulation represents the velocity predictions of a model based off of the feedforward gains from the tool (these are used to calculate the "sim" error metrics mentioned above).
 
-The quasistatic test ought to have nearly linear velocity, and nearly-zero acceleration (hense "quasistatic"). The dynamic test ought to have velocity that asymptotically approaches a steady-state speed (the shape of the curve should be exponential, in fact), and acceleration that, accordingly, rapidly falls to zero (also exponentially, as the derivative of an exponential function is also an exponential function).
+A successful quasistatic graph ought to be very nearly linear. A successful dynamic graph ought to have velocity that exponentially approaches a steady-state speed.
 
 Deviation from this behavior is a sign of an :ref:`error <docs/software/pathplanning/system-identification/viewing-diagnostics:Common Failure Modes>`, either in your robot setup, analysis settings, or your test procedure.
 
-Voltage-Domain Diagnostics
---------------------------
+Acceleration-Velocity Plot
+^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-The Voltage-Domain Diagnostics plots velocity and acceleration versus voltage. Velocity is ploted for the quasistatic test, and acceleration is plotted for the dynamic test. For a typical drive identification, the plots should resemble this (again, other mechanisms will be similar)
+The acceleration-versus-velocity plot displays the mechanism velocity versus the portion of acceleration corresponding to factors other than friction (ideally, this would leave only back-EMF) and applied voltage across *all* of the tests.
 
-.. image:: images/voltagedomain-plots.png
+.. image:: images/accel-vs-vel-plot.png
    :alt: Voltage domain plots picture
 
-Both plots should be linear, however the dynamic plot will almost certainly have substantially-more noise. The noise on the dynamic plot may be reduced by increasing the :guilabel:`Window Size` setting.
+This plot should be quite linear, with patches of relatively noiseless quasistatic data intermixed with quite-noisy dynamic data. The noise on the dynamic sections of the plot may be reduced by increasing the :guilabel:`Window Size` setting.
 
 .. image:: images/window-size-selector.png
    :alt: Picture of the window size selector
 
 However, if your robot or mechanism has low mass compared to the motor power, this may "eat" what little meaningful acceleration data you have (however, in these cases ``kA`` will tend towards zero and can usually be ignored, anyway).
```

Comment:
An accurate `kA` actually makes a huge difference if you're using LQR, especially for low-inertia mechanisms. If `kA` isn't accurate, the feedback analysis results can't be trusted.